src/certifier_tests.cc

```diff
@@ -38,158 +38,153 @@ DEFINE_string(ask_cert_file_name, "ask_cert_file.bin", "ask cert");
 DEFINE_string(vcek_cert_file_name, "vcek_cert_file.bin", "vcek cert");
 
 // Encryption and support tests
-extern bool test_random(bool print_all);
+
 TEST (test_random, test_random) {
   EXPECT_TRUE(test_random(FLAGS_print_all));
 }
-extern bool test_digest(bool print_all);
+
 TEST (test_digest, test_digest) {
   EXPECT_TRUE(test_digest(FLAGS_print_all));
 }
-extern bool test_encrypt(bool print_all);
+
 TEST (test_encrypt, test_encrypt) {
   EXPECT_TRUE(test_encrypt(FLAGS_print_all));
 }
-extern bool test_authenticated_encrypt(bool print_all);
+
 TEST (test_authenticated_encrypt, test_authenticated_encrypt) {
   EXPECT_TRUE(test_authenticated_encrypt(FLAGS_print_all));
 }
-extern bool test_public_keys(bool print_all);
+
 TEST (public_keys, test_public_keys) {
   EXPECT_TRUE(test_public_keys(FLAGS_print_all));
 }
-extern bool test_sign_and_verify(bool print_all);
+
 TEST (sign_and_verify, test_sign_and_verify) {
   EXPECT_TRUE(test_sign_and_verify(FLAGS_print_all));
 }
-extern bool test_key_translation(bool print_all);
+
 TEST (key_translation, test_key_translation) {
   EXPECT_TRUE(test_key_translation(FLAGS_print_all));
 }
-extern bool test_time(bool print_all);
+
 TEST (time, test_time) {
   EXPECT_TRUE(test_time(FLAGS_print_all));
 }
 
 // Basic Primitive tests
-extern bool test_seal(bool print_all);
 TEST (seal, test_seal) {
   EXPECT_TRUE(test_seal(FLAGS_print_all));
 }
-extern bool test_attest(bool print_all);
+
 TEST (attest, test_attest) {
   EXPECT_TRUE(test_attest(FLAGS_print_all));
 }
 
 // Admission Tests
-extern bool test_artifact(bool print_all);
+
 TEST (artifact, test_artifact) {
   EXPECT_TRUE(test_artifact(FLAGS_print_all));
 }
 
 // File protection and store tests
-extern bool test_protect(bool print_all);
 TEST (protect, test_protect) {
   EXPECT_TRUE(test_protect(FLAGS_print_all));
 }
-extern bool test_policy_store(bool print_all);
+
 TEST (policy_store, test_policy_store) {
   EXPECT_TRUE(test_policy_store(FLAGS_print_all));
 }
-extern bool test_init_and_recover_containers(bool print_all);
+
 TEST (init_and_recover_containers, test_init_and_recover_containers) {
   EXPECT_TRUE(test_init_and_recover_containers(FLAGS_print_all));
 }
 
 // policy tests
-extern bool test_claims_1(bool print_all);
 TEST (test_claims_1, test_claims_1) {
   EXPECT_TRUE(test_claims_1(FLAGS_print_all));
 }
-extern bool test_signed_claims(bool print_all);
+
 TEST (signed_claims, test_signed_claims) {
   EXPECT_TRUE(test_signed_claims(FLAGS_print_all));
 }
 
-extern bool test_local_certify(string&, bool, string&, string&);
+extern bool test__local_certify(string&, bool, string&, string&);
```

Comment:
This `test_local_certify()` and `test__local_certify()` jugglery is just so that we have the former available with the default signature (i.e., `(bool print_all)`), just like all the other test methods.

The former variant becomes the default test-case method, with default arguments, supplied to call the underlying latter method.

Then the former, default, test-case method gets called via the Python bindings from `test_certifier_tests.py()`